doc/variableexplorer.rst

```diff
@@ -2,95 +2,235 @@
 Variable Explorer
 #################
 
-The **Variable Explorer** shows the namespace contents (all global object references, such as variables, functions, modules, etc.) of the currently selected :doc:`ipythonconsole` session, and allows you to interact with them through a variety of GUI-based editors.
+The **Variable Explorer** allows you to interactively browse and manage the variables and objects generated running your code.
 
-.. image:: images/variable_explorer/variable_explorer_standard.png
+.. image:: images/variable_explorer/variable-explorer-execution.gif
+   :width: 500
    :align: center
-   :alt: Spyder Variable Explorer, with a list of variables and their contents
+   :alt: Spyder Variable Explorer execution with a variable of type list
 
 |
 
+It shows the namespace contents (including all global objects, variables, class instances and more) of the currently selected :doc:`ipythonconsole` session, and allows you to add, remove, and edit their values through a variety of GUI-based editors.
+
+.. image:: images/variable_explorer/variable-explorer-standard.png
+   :align: center
+   :alt: Spyder Variable Explorer, with a list of variables and their contents
 
-====================
-Features and editors
-====================
+|
 
-Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image.
-Several examples of this functionality follow:
+The Variable Explorer gives you information on the name, size, type and value of each object. To modify a scalar variable, like an number, string or boolean, simply double click it in the pane and type its new value.
+
+.. image:: images/variable_explorer/variable-explorer-modifying.gif
+   :align: center
+   :alt: Spyder Variable Explorer modifying value of a variable 
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_text_long.png
+==============
+Object Viewers
+==============
+
+Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, Series and more; as well as being able to plot and visualize them with one click.
+
+
+
+~~~~~~~
+Strings
+~~~~~~~
+
+When a string variable is longer than forty characters, you can double click it to see its value in a text editor to more easily modify it.
+
+.. image:: images/variable_explorer/variable-explorer-text-long.png
```

Comment:
This one has an error in the first line: "Recieve" must be "Receive". Also, I would only like to see the first line (i.e. not showing from `Parameters` down) to not be confused with the functionality of showing docstrings of the Object Explorer.

Reply:
But that defeats the point of the long text editor as its only one line, and furthermore is the exact opposite of what we both agreed to tell @juanis2112 to change on this screenshot that resulted in her redoing it to have more, and more "realistic", text. If we're going to change it, we need to come up with some other text to show here that is a similar size.

Reply:
Ok, but then we shouldn't show a docstring but another kind of text. Could you come up with a different text then?

Reply:
I fixed the one that was already there, if you come up with a better one let me know.

Reply:
I think its fine, but if @ccordoba12 insists on having something different, here's a random snippit from a CSV, that a user might conceivably read in as a string and want to parse later. Feel free to simplify as desired.

```
time,runtime,ping,adc_vb_f,adc_va_f,adc_vl_f,adc_ic_f,adc_il_f,t_hs,t_batt,t_amb,t_rts,charge_state,array_fault,vb_f,vb_ref,ahc_r,ahc_t,kwhc,load_state,load_fault,v_lvd,ahl_r,ahl_t,hourmeter,alarm,dip_switch,led_state,power_out,sweep_vmp,sweep_pmax,sweep_voc,vb_min_daily,vb_max_daily,ahc_daily,ahl_daily,array_fault_daily,load_fault_daily,alarm_daily,vb_min,vb_max,sequence_count,timestamp,crc_errors,valid_packets,bytes_read,bytes_written,bytes_remaining,packets_sent,packets_dropped
2019-11-13 00:00:29.797264,490800.0,0,12.509,0.656,12.485,0.0,1.164,0.0,1.0,1.0,25.0,3,0000000000000000,12.506,0.0,175.7,175.7,2.4,1,0000000000000000,11.484,145.8,145.8,7413,00000000000000000000000000000001,0000000000001101,6,0.0,8.435,7.172,11.249,12.469,12.527,0.0,0.6,0000000000000000,0000000000000000,00000000000000000000000000000001,9.439,19.47,231814,2019-11-13 14:36:23.389000,0,867,49871585280,19074114444,236913627136,0,0
2019-11-13 00:01:29.797262,490860.0,0,12.5,0.656,12.463,0.0,1.169,0.0,1.0,1.0,25.0,3,0000000000000000,12.503,0.0,175.7,175.7,2.4,1,0000000000000000,11.484,145.8,145.8,7413,00000000000000000000000000000001,0000000000001101,6,0.0,8.435,7.172,11.249,12.469,12.527,0.0,0.6,0000000000000000,0000000000000000,00000000000000000000000000000001,9.439,19.47,231874,2019-11-13 14:37:23.396000,0,867,49879449600,19074114444,236913627136,0,0
2019-11-13 00:02:29.797257,490920.0,0,12.497,0.656,12.463,0.0,1.167,0.0,1.0,1.0,25.0,3,0000000000000000,12.5,0.0,175.7,175.7,2.4,1,0000000000000000,11.484,145.8,145.8,7413,00000000000000000000000000000001,0000000000001101,6,0.0,8.435,7.172,11.249,12.469,12.527,0.0,0.6,0000000000000000,0000000000000000,00000000000000000000000000000001,9.439,19.47,231934,2019-11-13 14:38:23.405000,0,867,49887576064,19074114444,236913627136,0,0
2019-11-13 00:03:29.797264,490980.0,0,12.491,0.656,12.46,0.0,1.193,0.0,1.0,1.0,25.0,3,0000000000000000,12.494,0.0,175.7,175.7,2.4,1,0000000000000000,11.484,145.8,145.8,7413,00000000000000000000000000000001,0000000000001101,6,0.0,8.435,7.172,11.249,12.469,12.527,0.0,0.6,0000000000000000,0000000000000000,00000000000000000000000000000001,9.439,19.47,231994,2019-11-13 14:39:23.414000,0,867,49895440384,19074114444,236913627136,0,0
```

Reply:
Let's leave it as it is.